src/Main.ts

```diff
@@ -1615,6 +1617,22 @@ export class Main {
         await this.bridgeBlocker?.checkLimits(state.activeUsers);
     }
 
+    private async pingBridge() {
+        let internalRoom: string|null;
+        try {
+            internalRoom = await this.datastore.getUserAdminRoom("-internal-");
```

Comment:
Since the bridge admin rooms is optional in the config, I wanted to use a different room. It also reduces the amount of (hopefully invisible) message spam the user sees.